docs/deployment/infrastructure.md

```diff
@@ -194,7 +194,13 @@ Use the following shorthand for conveying the Resource Type as part of the Resou
 The following steps are required to set up the environment, with linked issues to automate them:
 
 - `terraform apply`
+- Set up Slack notifications
+  1. [Create a Slack email](https://slack.com/help/articles/206819278-Send-emails-to-Slack) for the [#benefits-notify](https://cal-itp.slack.com/archives/C022HHSEE3F) channel
+  1. [Set it as a Secret in the Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/secrets/quick-create-portal#add-a-secret-to-key-vault) named `slack-benefits-notify-email`
 - Set required [App Service configuration](../configuration/environment-variables.md)
+- [Upload configuration data](../configuration/data.md) to the storage container
+- [Set up webhook from GitHub](https://github.com/cal-itp/benefits/settings/hooks) to [App Service Deployment Center](https://learn.microsoft.com/en-us/azure/app-service/deploy-ci-cd-custom-container?tabs=acr&pivots=container-linux) for the `Packages` event
+- Add the default hostname (`https://<app service name>.azurewebsites.net`) to the reCAPTCHA configuration
```

Comment:
No, we don't want to do this.

Reply:
I want to keep reCAPTCHA listed here since this is useful for creating/rebuilding an environment… thoughts about how to word it? "Add the custom domain"?

Reply:
I'm not sure I agree - reCAPTCHA is configured to work on the existing "custom domains" `dev`, `test`, and prod. From the infra point of view, the task is to configure the infra to use the custom domains. There shouldn't be anything to do on the reCAPTCHA side.